docs/source/architecture/slashing.rst

```diff
@@ -26,16 +26,16 @@ The entire stake consists of:
     * tokens which the staker can withdraw at any moment
     * tokens locked for a specific period
 
-A staker may extend the unlock period for any number of portions of their total stake. This divides the stake into smaller parts, each with a unique unlock date in the future. Stakers may also acquire and lock new tokens. The total stake is represented as the sum of all the different sub-stakes active in a given cycle (new cycle every 24h), which includes locked sub-stakes, and any sub-stakes that have passed their unlock date, and can be freely withdrawn. Each sub-stake has a beginning and duration (lock time). When a staker confirms activity each day, the remaining lock time for relevant sub-stakes is reduced. 
+.. _`Sub-stakes`: https://docs.nucypher.com/en/latest/architecture/sub_stakes.html
 
-Sub stakes get slashed in the order of their remaining lock time, beginning with the shortest – so the first portion of the stake to be slashed is the unlocked portion. After that, if necessary, locked sub-stakes are decreased – the shortest sub stake is decreased by the required amount; if the adjustment of that sub-stake is insufficient to fulfil the required punishment sum, then the next shortest sub-stake is decreased, and so on. Sub-stakes that begin in the next period are checked separately.
+`Sub-stakes`_ get slashed in the order of their remaining lock time, beginning with the shortest – so the first portion of the stake to be slashed is the unlocked portion. After that, if necessary, locked sub-stakes are decreased – the shortest sub-stake is decreased by the required amount; if the adjustment of that sub-stake is insufficient to fulfil the required punishment sum, then the next shortest sub-stake is decreased, and so on. Sub-stakes that begin in the next period are checked separately.
 
 **Example:**
 
     A staker has 1000 tokens:
-        * 1st sub stake = 500 tokens locked for 10 periods
-        * 2nd sub stake = 200 tokens for 2 periods
-        * 3rd sub stake = 100 tokens locked starting from the next period and locked for 5 periods. The 3rd sub stake is locked for the next period but has not yet been used as a deposit for "work" - not until the next period begins.
+        * 1st sub-stake = 500 tokens locked for 10 periods
```

Comment:
Should we use "locking duration"...? here and in other spots in the document?
```suggestion
        * 1st sub-stake = 500 tokens with a locking duration of 10 periods
```

Reply:
What about this comment from @derekpierre ?

Reply:
Actually that's resolved, see one resolved comment above